CRM/Core/Config/MagicMerge.php

```diff
@@ -402,7 +402,6 @@ private function initLocals() {
         'inCiviCRM' => FALSE,
         'doNotResetCache' => 0,
         'keyDisable' => FALSE,
-        'initialized' => FALSE,
```

Comment:
There was some confusion in `MagicMerge.php` about whether`$config->initialized` maps to`$config->locals['initialized']` or `$runtime->initialized`. As things stand, it actually maps to `$runtime->initialized`, so this line is meaningless.

Removing it makes it easier to search for `initialized` and see what's happening.